UI/SidePanel.cpp

```diff
@@ -2530,6 +2545,10 @@ void SidePanel::PlanetPanelContainer::PlanetLeftClicked(int planet_id) {
     PlanetSelectedSignal(planet_id);
 }
 
+void SidePanel::PlanetPanelContainer::PlanetLeftDoubleClicked(int planet_id) {
+    PlanetLeftDoubleClickedSignal(planet_id);
```

Comment:
Move open/close braces to the same line as the details of a single-line function such as this...

{ PlanetLeftDoubleClickedSignal(planet_id); }